packages/components/src/utils/isExternalUrl.ts

```diff
@@ -0,0 +1,9 @@
+// Determine if the provided URL is an external link
+export const isExternalUrl = (url: string) => {
+  return (
+    url.startsWith("http://") ||
```

Comment:
actually there are a lot more schemes:
https://omni.wikiwand.com/en/articles/List_of_URI_schemes

Reply:
Good point, updated to use the internal link definition in d4e8aaea0c44ce5089c9d33c5f2076e68a548192.